sleap/gui/learning/runners.py

```diff
@@ -109,9 +109,9 @@ def cli_args(self):
 
         # -Y represents endpoint of [X, Y) range but inference cli expects
         # [X, Y-1] range (so add 1 since negative).
-        frame_int_list = [i + 1 if i < 0 else i for i in self.frames]
+        frame_int_set = set([i + 1 if i < 0 else i for i in self.frames])
```

Comment:
Do we maybe need to convert back to list and sort? Sets don't guarantee ordering.

Feel free to ignore if this is not an issue in this case.